src/components/EventForm/index.js

```diff
@@ -129,35 +137,73 @@ export default function EventForm({
       </div>
     </div>
   );
+  const accion=()=>{
+    onFinish(form.getFieldsValue());
+  }
+  
+  const onInput = (e) => {
+    //console.log(e.target.value)
+    setValueTitle(e.target.value)};
+
+  const onChange = (date, dateString) => {
+      //console.log(dateString);
+      if(dateString !== ""){
+        setValueDate(dateString)
+      }
+  };
+
 
   return (
     <Modal
       open={isModalOpen}
-      onOk={() => {
-        onFinish(form.getFieldsValue());
-      }}
+      onOk={()=>{accion}}
       onCancel={() => {
         closeModal();
       }}
-      okText={"Save"}
       confirmLoading={isLoading}
       centered
+      footer={[
+        <Button onClick={()=>{closeModal()}}>Cancel</Button>,
+        <Button type="primary" onClick={accion} disabled={valueDate !== "" && valueTitle !== "" ? false : true} >Save</Button>,
+        <br></br>,
+        <Tooltip placement="topLeft" title="Save available when title and date are filled in" >
+          <InfoCircleOutlined />
+        </Tooltip>
+      ]}
+     
     >
       <Form
-        initialValues={event ? { ...event, date: moment.utc(event.date) } : {}}
+        initialValues={event ? { ...event, date: moment.utc(event.date), requiredMarkValue: requiredMark } 
+        : {requiredMarkValue: requiredMark}}
         layout={"vertical"}
         form={form}
-        onValuesChange={() => {}}
+        onValuesChange={() => {onRequiredTypeChange}}
```

Comment:
Why give an option to toggle the required type? These were not in the pipeline. also feels unnecessary.

Passing a function inside a callback is incorrect btw